pkg/webui/lib/shared-messages.js

```diff
@@ -115,6 +116,7 @@ export default defineMessages({
   connecting: 'Connecting',
   connectionIssues: 'Connection issues',
   contactInformation: 'Contact information',
+  contactFieldPlaceholder: 'Type to choose a contact',
```

Comment:
```suggestion
  contactFieldPlaceholder: 'Type or choose a collaborator',
```
Diverting from the wireframe here as I think `collaborator` is more fitting.